pkgs/tools/networking/network-manager/applet/default.nix

```diff
@@ -81,7 +79,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     homepage = "https://gitlab.gnome.org/GNOME/network-manager-applet/";
     description = "NetworkManager control applet for GNOME";
-    license = licenses.gpl2;
+    license = licenses.gpl2Only;
```

Comment:
It is necessary to look into the source code: https://gitlab.gnome.org/GNOME/network-manager-applet/-/blob/74a71cbe52c961701d4d8b70bcf76fa0ed83a5d3/meson.build#L4

The license identified by GitHub/GitLab is misleading. See also https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206.

Reply:
Ack, thanks for noting.